apps/dashboard/test/models/motd_formatter_markdown_erb_test.rb

```diff
@@ -0,0 +1,38 @@
+require 'test_helper'
+
+class MotdTest < ActiveSupport::TestCase
+
+  test "test when motd_formatter_markdown_erb_valid" do
+
+    path = "#{Rails.root}/test/fixtures/files/motd_valid"
+    motd_file = MotdFile.new(path)
+    formatted_motd = MotdFormatterMarkdownErb.new(motd_file)
+    expected_file = OodAppkit.markdown.render(motd_file.content)
+
+    assert_equal expected_file, formatted_motd.content
+  end
+
+  test "test when motd_formatter_markdown_erb_empty" do
+    path = "#{Rails.root}/test/fixtures/files/motd_empty"
+    motd_file = MotdFile.new(path)
+    formatted_motd = MotdFormatterMarkdownErb.new(motd_file)
+
+    assert_equal '', formatted_motd.content
+  end
+
+  test "test when motd_formatter_markdown_erb_missing" do
+    path = "#{Rails.root}/test/fixtures/files/motd_missing"
+    motd_file = MotdFile.new(path)
+    formatted_motd = MotdFormatterMarkdownErb.new(motd_file)
+
+    assert_equal '', formatted_motd.content
+  end
+
+  test "test when motd_formatter_markdown_erb_nil" do
+    motd_file = nil
+    formatted_motd = MotdFormatterMarkdownErb.new(motd_file)
+
+    assert_not_nil formatted_motd.content
+  end
+end
```

Comment:
I think we'd want more test cases, specifically thses.

1.  renders ERB correctly
2.  renders plain text correctly (the output's the same as the other MotcFormatter when input has no ERB)
3.  gracefully handles exceptions

We don't need super large files as input files examples like the other test cases, just something that has _some_ ruby. Also you can have the expected output as a fixture file too for readability.